discrete_env.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from gym import Env, spaces
```

Comment:
According to Mo, avoid statements like `from foo import bar`. It's better to use the fully qualified function name like `foo.bar` so it's easier to read.